### PR TITLE
[VAS] Story 10917: Update nginx configuration.

### PR DESCRIPTION
**Pull request by GiooDev**
## Description

* Centralize logs and configuration under /vitamui/log/nginx/.

## Type de changement:

* Ansiblerie

## Contributeur

* VAS (Vitam Accessible en Service)

### CODE REVIEW

File: deployment/roles/reverse/templates/nginx/conf.d/vhosts.conf.j2 [hunk 1]
```diff
@@ -2,11 +2,15 @@ include /etc/nginx/conf.d/upstream;
 
 server {
 
-  listen 443 ssl;
+  listen {{ ip_service }}:443 ssl;
   include {{ nginx_ssl_conf }}/ssl.conf;
+  include /etc/nginx/conf.d/proxy_params;
```
Comment (ghost) [inline deployment/roles/reverse/templates/nginx/conf.d/vhosts.conf.j2]: Attention à ne pas forcément le commoniser.

Ce n'étais pas présent pour le bloc cas :

```
  location = /cas {
    return 302 $scheme://$host/cas/login;
  }
```

En particulier, l'ajout & suppression des headers type `Authorization` & `X-Forwarded-For` n'affectent-ils pas l'authentification sur cas ou les logs?

File: deployment/roles/reverse/templates/nginx/conf.d/vhosts.conf.j2 [hunk 2]
```diff
@@ -2,11 +2,14 @@ include /etc/nginx/conf.d/upstream;
 
 server {
 
-  listen 443 ssl;
+  listen {{ ip_service }}:443 ssl;
   include {{ nginx_ssl_conf }}/ssl.conf;
 
   server_name {{ vitam_reverse_external_dns }};
 
+  access_log  {{ vitamui_defaults.folder.root_path | default('/vitamui') }}/log/nginx/access.log main;
```
Comment (GiooDev) [inline deployment/roles/reverse/templates/nginx/conf.d/vhosts.conf.j2]: Prévoir un logrotate pour ces logs.

Comment (GiooDev) [reply deployment/roles/reverse/templates/nginx/conf.d/vhosts.conf.j2]: Et créer les répertoires associés pour ces logs sinon le service ne démarre pas.

**Comment by TDevillechabrolle** [comment]
Logo
**Checkmarx One – Scan Summary & Details** – [9ea7efdd\-a35b\-4ccd\-992b\-ed434f1c5c78](https://eu.ast.checkmarx.net/projects/78091106-f137-4b7b-8b68-77ee0f4d3433/scans?id=9ea7efdd-a35b-4ccd-992b-ed434f1c5c78&branch=story_10917_secure_nginx)
### New Issues
Severity|Issue|Source File / Package|Checkmarx Insight
---|---|---|---
MEDIUM|[Privacy_Violation](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/359/12956636075206043460)|/api/api\-iam/iam\-security/src/main/java/fr/gouv/vitamui/iam/security/service/InternalSecurityService\.java: [110](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//api/api-iam/iam-security/src/main/java/fr/gouv/vitamui/iam/security/service/InternalSecurityService.java#L110)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=YOttuugejtALHlyj5g0e4BVjW%2Bk%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/ui/ui\-identity/src/main/java/fr/gouv/vitamui/identity/rest/CustomerController\.java: [194](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//ui/ui-identity/src/main/java/fr/gouv/vitamui/identity/rest/CustomerController.java#L194)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=VhYn7FOn1Zb36zIp8JIRHvqWcSk%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/ui/ui\-identity/src/main/java/fr/gouv/vitamui/identity/rest/CustomerController\.java: [194](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//ui/ui-identity/src/main/java/fr/gouv/vitamui/identity/rest/CustomerController.java#L194)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=84GMIiBmw8hG1ITnWVnjhBH6lcE%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/ui/ui\-identity/src/main/java/fr/gouv/vitamui/identity/rest/CustomerController\.java: [194](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//ui/ui-identity/src/main/java/fr/gouv/vitamui/identity/rest/CustomerController.java#L194)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=yAs%2FYO91rtOmYbTazYMoF5SgeE8%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/ui/ui\-identity/src/main/java/fr/gouv/vitamui/identity/rest/CustomerController\.java: [114](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//ui/ui-identity/src/main/java/fr/gouv/vitamui/identity/rest/CustomerController.java#L114)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=4wOnKXra%2BjTYiFyp1eEdiDrm%2FpU%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/ui/ui\-identity/src/main/java/fr/gouv/vitamui/identity/rest/CustomerController\.java: [114](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//ui/ui-identity/src/main/java/fr/gouv/vitamui/identity/rest/CustomerController.java#L114)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=lSRHYvKWhNceVUzthaa0xcNBiTU%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/api/api\-collect/collect\-external/src/main/java/fr/gouv/vitamui/collect/external/server/rest/SearchCriteriaHistoryExternalController\.java: [94](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//api/api-collect/collect-external/src/main/java/fr/gouv/vitamui/collect/external/server/rest/SearchCriteriaHistoryExternalController.java#L94)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=NgTPTAdsAtXJYQrIh%2FdhTG%2BKwWQ%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/api/api\-referential/referential\-external/src/main/java/fr/gouv/vitamui/referential/external/server/rest/OntologyExternalController\.java: [162](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//api/api-referential/referential-external/src/main/java/fr/gouv/vitamui/referential/external/server/rest/OntologyExternalController.java#L162)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=kG2y%2F0W4xwv73nC2OQS7Tpu8qI4%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/api/api\-archive\-search/archive\-search\-external/src/main/java/fr/gouv/vitamui/archives/search/external/server/rest/SearchCriteriaHistoryExternalController\.java: [108](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//api/api-archive-search/archive-search-external/src/main/java/fr/gouv/vitamui/archives/search/external/server/rest/SearchCriteriaHistoryExternalController.java#L108)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=a9h97LWkI0ranFd%2F6mFFJPrZT5g%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/ui/ui\-archive\-search/src/main/java/fr/gouv/vitamui/archives/search/rest/SearchCriteriaHistoryController\.java: [106](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//ui/ui-archive-search/src/main/java/fr/gouv/vitamui/archives/search/rest/SearchCriteriaHistoryController.java#L106)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=k5iT7tuIIzXaGX8LJLf5%2FBJuWP4%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/api/api\-iam/iam\-external/src/main/java/fr/gouv/vitamui/iam/external/server/rest/SubrogationExternalController\.java: [189](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//api/api-iam/iam-external/src/main/java/fr/gouv/vitamui/iam/external/server/rest/SubrogationExternalController.java#L189)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=C2vpG1%2FCVaAhfz35A%2Flth62Xx7M%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/ui/ui\-referential/src/main/java/fr/gouv/vitamui/referential/rest/OntologyController\.java: [191](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//ui/ui-referential/src/main/java/fr/gouv/vitamui/referential/rest/OntologyController.java#L191)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=rL2vtg8SghIJKIS6AdBG01uL1jM%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/ui/ui\-referential/src/main/java/fr/gouv/vitamui/referential/rest/SecurityProfileController\.java: [190](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//ui/ui-referential/src/main/java/fr/gouv/vitamui/referential/rest/SecurityProfileController.java#L190)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=qaxdL39S%2BWl1l7gNopt0eUYe%2Bto%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/api/api\-referential/referential\-external/src/main/java/fr/gouv/vitamui/referential/external/server/rest/SecurityProfileExternalController\.java: [160](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//api/api-referential/referential-external/src/main/java/fr/gouv/vitamui/referential/external/server/rest/SecurityProfileExternalController.java#L160)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=WRUyvm8AvQWK9JDnAg9EQ5jm4ZY%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/ui/ui\-collect/src/main/java/fr/gouv/vitamui/collect/rest/ProjectArchiveUnitController\.java: [106](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//ui/ui-collect/src/main/java/fr/gouv/vitamui/collect/rest/ProjectArchiveUnitController.java#L106)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=1%2Fva0ss8lCBoH%2FisyFx7zdf5aDE%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/ui/ui\-identity/src/main/java/fr/gouv/vitamui/identity/rest/CustomerController\.java: [114](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//ui/ui-identity/src/main/java/fr/gouv/vitamui/identity/rest/CustomerController.java#L114)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=HjlY6KvC7xV4qtSXqEk%2FmaWhkZM%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/api/api\-referential/referential\-external/src/main/java/fr/gouv/vitamui/referential/external/server/rest/FileFormatExternalController\.java: [175](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//api/api-referential/referential-external/src/main/java/fr/gouv/vitamui/referential/external/server/rest/FileFormatExternalController.java#L175)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=wPuK6IKJTYMkffd1vrt6FWSvgCk%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/ui/ui\-referential/src/main/java/fr/gouv/vitamui/referential/rest/FileFormatController\.java: [219](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//ui/ui-referential/src/main/java/fr/gouv/vitamui/referential/rest/FileFormatController.java#L219)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=x4hHgZePioOy4dNCTUms3riNBek%3D&redirect=true)

### Fixed Issues
Severity|Issue|Source File / Package|Checkmarx Insight
---|---|---|---
MEDIUM|[Privacy_Violation](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/359/12956636075206043460)|/api/api\-iam/iam\-security/src/main/java/fr/gouv/vitamui/iam/security/service/InternalSecurityService\.java: [110](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//api/api-iam/iam-security/src/main/java/fr/gouv/vitamui/iam/security/service/InternalSecurityService.java#L110)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=HLA1cKvcPqDbBe%2BA%2BnmzgzbtvvA%3D&redirect=true)
MEDIUM|[Privacy_Violation](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/359/12956636075206043460)|/api/api\-iam/iam\-security/src/main/java/fr/gouv/vitamui/iam/security/service/InternalSecurityService\.java: [110](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//api/api-iam/iam-security/src/main/java/fr/gouv/vitamui/iam/security/service/InternalSecurityService.java#L110)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=KNcuHxMYiE01w7dwnSf4J%2BnRRpE%3D&redirect=true)
MEDIUM|[Privacy_Violation](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/359/12956636075206043460)|/api/api\-iam/iam\-security/src/main/java/fr/gouv/vitamui/iam/security/service/InternalSecurityService\.java: [110](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//api/api-iam/iam-security/src/main/java/fr/gouv/vitamui/iam/security/service/InternalSecurityService.java#L110)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=hVL4Q3SSl8KRBpJ7D5GcKnnVTkI%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/api/api\-iam/iam\-internal/src/main/java/fr/gouv/vitamui/iam/internal/server/rest/CustomerInternalController\.java: [198](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//api/api-iam/iam-internal/src/main/java/fr/gouv/vitamui/iam/internal/server/rest/CustomerInternalController.java#L198)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=KL%2BOl2p0LHG8Q046HkwOhAQcrTo%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/api/api\-iam/iam\-internal/src/main/java/fr/gouv/vitamui/iam/internal/server/rest/CustomerInternalController\.java: [198](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//api/api-iam/iam-internal/src/main/java/fr/gouv/vitamui/iam/internal/server/rest/CustomerInternalController.java#L198)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=xjO95QTDxirAHJf8rEKQecca%2B8k%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/api/api\-iam/iam\-internal/src/main/java/fr/gouv/vitamui/iam/internal/server/rest/CustomerInternalController\.java: [248](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//api/api-iam/iam-internal/src/main/java/fr/gouv/vitamui/iam/internal/server/rest/CustomerInternalController.java#L248)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=Xxl0uT8%2BsKiw28pQDPXa%2FOSKsV0%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/ui/ui\-identity/src/main/java/fr/gouv/vitamui/identity/rest/CustomerController\.java: [220](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//ui/ui-identity/src/main/java/fr/gouv/vitamui/identity/rest/CustomerController.java#L220)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=so9QdJXuNsNOb1pKhDYUNAE90q8%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/api/api\-iam/iam\-external/src/main/java/fr/gouv/vitamui/iam/external/server/rest/CustomerExternalController\.java: [244](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//api/api-iam/iam-external/src/main/java/fr/gouv/vitamui/iam/external/server/rest/CustomerExternalController.java#L244)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=a%2Bi052I0tM5YHizclJICNKBs%2Fh4%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/api/api\-iam/iam\-internal/src/main/java/fr/gouv/vitamui/iam/internal/server/rest/CustomerInternalController\.java: [248](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//api/api-iam/iam-internal/src/main/java/fr/gouv/vitamui/iam/internal/server/rest/CustomerInternalController.java#L248)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=Ob5r9twwnsu71qKZ0i4qyhcmKUQ%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/ui/ui\-identity/src/main/java/fr/gouv/vitamui/identity/rest/CustomerController\.java: [220](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//ui/ui-identity/src/main/java/fr/gouv/vitamui/identity/rest/CustomerController.java#L220)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=AFjZhh6aWxsZH7p6MP%2F%2Bt0jnJM0%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/api/api\-iam/iam\-external/src/main/java/fr/gouv/vitamui/iam/external/server/rest/CustomerExternalController\.java: [244](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//api/api-iam/iam-external/src/main/java/fr/gouv/vitamui/iam/external/server/rest/CustomerExternalController.java#L244)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=%2B2Ru1XOTJ%2BeQbBMtEqMz8%2BGrxos%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/api/api\-archive\-search/archive\-search\-internal/src/main/java/fr/gouv/vitamui/archive/internal/server/rest/ArchiveSearchInternalController\.java: [135](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//api/api-archive-search/archive-search-internal/src/main/java/fr/gouv/vitamui/archive/internal/server/rest/ArchiveSearchInternalController.java#L135)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=pFaeJmhXWc%2BVE%2BZuQT2gQ1O%2BUHc%3D&redirect=true)
LOW|[Log_Forging](https://eu.ast.checkmarx.net/results/78091106-f137-4b7b-8b68-77ee0f4d3433/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/sast/description/117/9884244256835554923)|/ui/ui\-collect/src/main/java/fr/gouv/vitamui/collect/rest/TransactionController\.java: [242](https://github.com/ProgrammeVitam/vitam-ui/blob/story_10917_secure_nginx//ui/ui-collect/src/main/java/fr/gouv/vitamui/collect/rest/TransactionController.java#L242)|[Attack Vector](https://eu.ast.checkmarx.net/results/9ea7efdd-a35b-4ccd-992b-ed434f1c5c78/78091106-f137-4b7b-8b68-77ee0f4d3433/sast?result-id=2AUZvM0sDgHpPLGmpFdoyhmGhAI%3D&redirect=true)

**Comment by GiooDev** [comment]
Close as it's already take into account in newest PR.

**Pull request closed without merging**